www/lendingbot.js

```diff
@@ -246,6 +258,8 @@ function Timespan(name, multiplier) {
             }
             if (currency == "BTC") {
                 return displayUnit.formatValue(earnings) + " <span class=" + currencyClass + ">" + displayUnit.name + "</span> / " + name + "<br/>"
+            } else if (currency == "USD" || currency == "USDT") {
+                return printFloat(earnings, 2) + " <span class=" + currencyClass + ">" + currency + "</span> / "+  name + "<br/>";
```

Comment:
Suggestion: I also add EUR, but maintain consistency with the surrounding code. Then in a separate PR we decide how to handle this particular problem.